docker/docker-compose.yml

```diff
@@ -28,6 +28,22 @@ services:
       - AWS_SECRET_ACCESS_KEY
       - AWS_REGION
 
+  mock-mhs-adaptor:
+    build:
+      context: ../
+      dockerfile: docker/mock-mhs-adaptor/Dockerfile
+    depends_on: 
+      - gp2gp
+    ports:
+      - "8081:8081"
+    environment:
+      - GP2GP_SERVER_PORT=8081
```

Comment:
Do we need this variable, doesn't seem to be used anywhere?

Reply:
In you're code you have 

MHS_MOCK_SERVER_PORT

maybe change anything used to MHS_MOCK_XXX

Reply:
Agreed MOCK_MHS_SERVER_PORT and MOCK_MHS_LOGGING_LEVEL.

The others still stay as GP2GP_AMQP... since they will be the same exact values as the adaptor. Makes it easier to dev/deploy.